crates/sui-graphql-rpc/src/types/event.rs

```diff
@@ -46,6 +49,10 @@ pub(crate) struct EventKey {
 
     /// Event Sequence Number
     e: u64,
+
+    /// The checkpoint_sequence_number at which this was viewed at, or `None` if the data was
+    /// requested at the latest checkpoint.
+    checkpoint_viewed_at: Option<u64>,
```

Comment:
nit: small name please! (`c`), either using `#[serde(rename = "c")]` or by changing the field name to match the style of the others in this cursor.

Reply:
addressed in https://github.com/MystenLabs/sui/pull/15996